premake5.lua

```diff
@@ -211,6 +211,12 @@ workspace("xenia")
     architecture("x86_64")
     if os.istarget("linux") then
       platforms({"Linux"})
+    elseif os.istarget("macosx") then
+      platforms({"Mac"})
+      xcodebuildsettings({           
+        ["GCC_WARN_UNUSED_VARIABLE"] = "NO",
```

Comment:
Is this an issue with something like unused arguments of virtual function overrides primarily (or preprocessor conditionals or assertions), or do we have a large number of truly unused variables?

Reply:
Yes we have those, i believe they crept in because the warning was disabled for a number of valid cases…